packages/@jsii/kernel/src/tar-cache/default-cache-root.ts

```diff
@@ -0,0 +1,28 @@
+import { tmpdir } from 'os';
+import { join } from 'path';
+
+export function defaultCacheRoot(): string {
+  switch (process.platform) {
+    case 'darwin':
+      if (process.env.HOME)
+        return join(
+          process.env.HOME,
+          'Library',
+          'Caches',
+          'com.amazonaws.jsii',
+          'package-cache',
+        );
```

Comment:
[nitpick] Line went over and got wrapped?
the other `join`s are single-line.

Reply:
Yeah, this one is just a bit longer and was wrapped by `prettier`. Indentation/formatting is entirely prettify-driven in the jsii repository.